### PR TITLE
[Ready] EosStation

### PR DESCRIPTION
**Pull request by EIB0N**
## About The Pull Request
Implementing a new map called "EosStation". It is a completely updated cerestation.

## Why It's Good For The Game
New Map, new interactions , mostly working mules, unique cargo shuttle, for much larger population of 80+, space exploration

## Changelog
:cl:
add: EosStation, Eos cargo shuttle; Have Fun!
/:cl:

### Full
EosStation_Full-min (2)

### Areas
EosStation_Area-min (1)

### Labour Shuttle
EosStation_Labour

### Emergency Shuttle
EosStation_Emergency_Shuttle

### Arrival
EosStation_Arrival

### Evac
EosStation_Evac

### Medbay
EosStation_Medbay

### Supply
EosStation_Supply

### AI
EosStation_AI

### Engineering
EosStation_Engineering

### Science
EosStation_Science

### Service
EosStation_Service

### Security, Command
EosStation_Sec_Command

EosStation_Service_main_asteroid

### Cargo Shuttle
EosStation_Cargo_Shuttle

EosStation_Cargo_Shuttle2

### CODE REVIEW

**Comment by notfrying1pans** [comment]
why hello fellow cerestation enjoyer

**Comment by remove32** [comment]
jesus fuckin christ

**Comment by EIB0N** [comment]
> jesus fuckin christ

I know, ill fix it

**Comment by EIB0N** [comment]
Fixed it!

**Comment by remove32** [comment]
im scared of eosstation

**Comment by EIB0N** [comment]
What are you scared of brother

**Comment by MMMiracles** [comment]
what did you do to my baby

**Comment by LemonInTheDark** [comment]
Hello, it's you from the future when this is used with working atmos at high pop.
Use more airlocks to prevent gas/cold spread 4head

**Comment by Fikou** [comment]
poor miracles...

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -576,6 +576,10 @@
 	suffix = "helio"
 	name = "cargo ferry (Helio)"
 
+/datum/map_template/shuttle/cargo/eos
+		suffix = "eos"
```
Comment (Fikou) [inline code/datums/shuttles.dm:580]: double indent

**Review by Paxilmaniac — COMMENTED** [review]
While I am unaware of the balancing on fulp, there are some mildly concerning maintenance loot spawns.

Like this 357 speedloader:
image

Or this shotgun:
image

How about this energy sword and nuke op submachinegun:
image

OR what about this room here that's completely full of techshells and a literal fucking captain's egun:
image

You know you probably shouldn't be filling maint with 15+ force weapons:
image

You get the idea, and those are just the worst offenders. Now, I'm no expert on the loot balancing of fulp, as I've never played on it, but don't you think this is a *tad* too much?

**Comment by EIB0N** [comment]
The e-swords are fake and so is the submachine gun, forgot to change the shotgun but changed to a fake riot shotgun the antique energy gun is to allow the prisoners a way to escape via a different method (fighting the legion and the clown) if you can provide an alternative besides a disabler please feel free. The speed reloader is for the russian revolver in the same room. The daggers are changed for a butcher's knife

**Comment by EIB0N** [comment]
> Hello, it's you from the future when this is used with working atmos at high pop.
> Use more airlocks to prevent gas/cold spread 4head
> 
> Edit: Jesus christ the rwalls

Can you explain what you mean a bit more

**Comment by EIB0N** [comment]
> While I am unaware of the balancing on fulp, there are some mildly concerning maintenance loot spawns.
> 
> Like this 357 speedloader:
> image
> 
> Or this shotgun:
> image
> 
> How about this energy sword and nuke op submachinegun:
> image
> 
> OR what about this room here that's completely full of techshells and a literal fucking captain's egun:
> image
> 
> You know you probably shouldn't be filling maint with 15+ force weapons:
> image
> 
> You get the idea, and those are just the worst offenders. Now, I'm no expert on the loot balancing of fulp, as I've never played on it, but don't you think this is a _tad_ too much?

The e-swords are fake and so is the submachine gun, forgot to change the shotgun but changed to a fake riot shotgun the antique energy gun is to allow the prisoners a way to escape via a different method (fighting the legion and the clown) if you can provide an alternative besides a disabler please feel free. The speed reloader is for the russian revolver in the same room. The daggers are changed for a single butcher's knife

**Comment by EIB0N** [comment]
> what did you do to my baby

Was cere your original idea?

**Comment by remove32** [comment]
> 
> 
> what did you do to my baby

TG is too cowardly to re-add this beauty. fulp will do it ourselves (and suffer the consequences)

**Comment by remove32** [comment]
> 
> 
> > Hello, it's you from the future when this is used with working atmos at high pop.
> > Use more airlocks to prevent gas/cold spread 4head
> > Edit: Jesus christ the rwalls
> 
> Can you explain what you mean a bit more

you should add more firelocks around the station to better segment air zones. also, you seem to have used reinforced walls as the base wall type which is not a good idea.

**Comment by Fikou** [comment]
> > what did you do to my baby
> 
> TG is too cowardly to re-add this beauty. fulp will do it ourselves (and suffer the consequences)

this isnt the beauty though, every department is completely redesigned to be worse and soulless

**Comment by EIB0N** [comment]
> > > what did you do to my baby
> > 
> > 
> > TG is too cowardly to re-add this beauty. fulp will do it ourselves (and suffer the consequences)
> 
> this isnt the beauty though, every department is completely redesigned to be worse and soulless

It's my first station it is very much subject to change this is just the finished version with no testing from a large amount of players also used the mentors to help me decide what they want since they pump on the hours

**Comment by EIB0N** [comment]
> > > Hello, it's you from the future when this is used with working atmos at high pop.
> > > Use more airlocks to prevent gas/cold spread 4head
> > > Edit: Jesus christ the rwalls
> > 
> > 
> > Can you explain what you mean a bit more
> 
> you should add more firelocks around the station to better segment air zones. also, you seem to have used reinforced walls as the base wall type which is not a good idea.

I used them a lot for stuff like maint due to the idea of the rock on the outside being easy to break. Aside from that just being unable to breach a department too easily.Send some examples of redundant walls so i can get a general idea of which I'd want to change

**Comment by LemonInTheDark** [comment]
> 
> 
> > Hello, it's you from the future when this is used with working atmos at high pop.
> > Use more airlocks to prevent gas/cold spread 4head
> > Edit: Jesus christ the rwalls
> 
> Can you explain what you mean a bit more

Use airlocks/firelocks to seal off spaces from each other. Seek to emulate box over delta, you don't want 1 airlock separating medbay and arrivals

**Comment by EIB0N** [comment]
> > > Hello, it's you from the future when this is used with working atmos at high pop.
> > > Use more airlocks to prevent gas/cold spread 4head
> > > Edit: Jesus christ the rwalls
> > 
> > 
> > Can you explain what you mean a bit more
> 
> Use airlocks/firelocks to seal off spaces from each other. Seek to emulate box over delta, you don't want 1 airlock separating medbay and arrivals
> 
> Edit: I think you get the rwalls thing, planned weakness is more interesting then unintentional impenetrability

I noticed the rwalls you guys are talking about and have fixed it, next Ill do the firelocks and ill repost the pictures so you guys can decide if I did it correctly

**Comment by EIB0N** [comment]
Ok the pictures have been updated

**Comment by EIB0N** [comment]
So due to my recent pr in the helio maint improvement ive decided that I want to make it similar so this will mean I will be making rather large maint sections in each part except arrival. Additionally the exploring part will be strictly for antags and some exploration but mostly for antags. Due to the large amount of area ill be removing the more aesthetic stuff in the map so the server doesn't die.

**Comment by Helianthus00** [comment]
You must have messed up the rebase or something. I recommend just remaking the pr with a fresh branch.

**Comment by JohnFulpWillard** [comment]
This PR would've crashed mapdiffbot if we werent still blacklisted. Just saying. It's best to start on a new branch.

**Comment by Helianthus00** [comment]
Closing to prevent crashing of mapdiffbot

**Pull request closed without merging**